nucliadb_client/nucliadb_client/resource.py

```diff
@@ -166,15 +166,15 @@ def add_text(
             fcmw.metadata.split_metadata[split].paragraphs.append(
                 Paragraph(
                     start=0,
-                    end=len(text),
+                    end=len(text) - 1,
```

Comment:
this one IMO is no big deal, as we are not using it that much and we should probably move client outside of the nucliadb repo